client-course-schedulizer/.eslintrc.js

```diff
@@ -7,9 +7,9 @@ module.exports = {
   },
   extends: [
     "react-app",
-    "airbnb",
```

Comment:
ESLint was causing errors. It's very outdated so for now, we'll work without a full eslint config.